src/Compilers/CSharp/Portable/Symbols/Retargeting/RetargetingSymbolTranslator.cs

```diff
@@ -805,7 +805,10 @@ public ImmutableArray<Symbol> Retarget(ImmutableArray<Symbol> arr)
 
                 foreach (var s in arr)
                 {
-                    symbols.Add(Retarget(s));
+                    if (!(s is TupleErrorFieldSymbol))
```

Comment:
>if (!(s is TupleErrorFieldSymbol)) [](start = 20, length = 34)

I think that instead of doing filtering here and making changes to compensate for it in RetargetingNamedTypeSymbol.cs we should change an implementation of ```RetargetingModuleSymbol.CreateRetargetingField``` to specially handle ```TupleErrorFieldSymbol``` by creating appropriate new instances of ```TupleErrorFieldSymbol``` instead of RetargetingFieldSymbols.